### ISSUE TITLE
The selector all predict False

### ISSUE THREAD

**Issue by duyuwen-duen** [bytedance/pasa#40]
Even though my training loss drops significantly, when I perform inference — even on the training set — the selector still outputs probabilities all below 0.5. Why is that?

![Image](https://github.com/user-attachments/assets/7fc6c711-e65a-43cb-9304-26f4d46f9856)
